mne/channels/montage.py

```diff
@@ -717,6 +717,30 @@ def save(self, fname):
                                'head coordinates.')
         write_dig(fname, self.dig)
 
+    def __getitem__(self, key):  # XXX: required for concat
+        raise NotImplementedError
+
+    def __iadd__(self, other):
+        raise NotImplementedError
+
+    # XXX: this is not used by `reduce(concat, list_of_DigMontages)`
+    # def __concat__(self, other): 
+    #     raise NotImplementedError
+
+    def __add__(self, other):
+        foo = deepcopy(self)
+        if foo.ch_names is None:
+            foo.ch_names = other.ch_names
+        elif other.ch_names is not None:
+            foo.ch_names += other.ch_names  # XXX: This is gona be complicated 'cos ch_names is sorted and in sync with foo.dig
```

Comment:
it's no problem as you keep the order with `foo.dig += other.dig`

I don't see the issue

Reply:
the issue is taht ch_names is ordered